packages/manager/modules/octavia-load-balancer/src/loadbalancers/loadbalancers.controller.js

```diff
@@ -0,0 +1,51 @@
+import { TRACKING_OCTAVIA_LOAD_BALANCERS_PREFIX } from './loadbalancers.constants';
+
+export default class OctaviaLoadBalancersCtrl {
+  /* @ngInject */
+  constructor(atInternet, Alerter, $translate) {
+    this.atInternet = atInternet;
+    this.Alerter = Alerter;
+    this.$translate = $translate;
+  }
+
+  $postLink() {
+    if (this.loadBalancerDeleteSuccess) {
+      this.Alerter.set(
+        'alert-info',
+        this.$translate.instant('octavia_load_balancer_delete_success'),
+        null,
+        'octavia.alerts.global',
+      );
+    }
+  }
+
+  createLoadBalancer() {
+    this.atInternet.trackClick({
+      name: `PCI_PROJECTS_LOAD_BALANCER_ADD`,
```

Comment:
I think this tracking name is not correct. To ask to requester

Reply:
Done